lighthouse/datadog_checks/lighthouse/data/conf.yaml.example

```
@@ -8,17 +8,32 @@ instances:
     #
   - min_collection_interval: 60
 
-    ## @param url - string - required
-    ## url of the website to get lighthouse metrics of
+    ## @param url - string - optional
+    ## url of the website to get lighthouse metrics of. This option is deprecated.
     ## ex: "https://www.datadoghq.com"
     #
     url: <URL_PLACEHOLDER>
```

```suggestion
    ## @param url - string - optional
    ## url of the website to get lighthouse metrics of. This option is deprecated.
    ## ex: "https://www.datadoghq.com"
    #
    # url: <URL_PLACEHOLDER>
```
Let's comment this out since it is now deprecated